docs/index.md

```diff
@@ -23,6 +23,12 @@ To install `pixi` you can run the following command in your terminal:
     curl -fsSL https://pixi.sh/install.sh | sh
     ```
 
+    If your system doesn't have `curl`, you can use `wget`:
+
+    ```bash
+    wget -qO- https://pixi.sh/install.sh | sh
+    ```
```

Comment:
Let's not add it to the main page, but the installation page: https://pixi.sh/latest/advanced/installation/